### DISCUSSION TITLE
How was the overflow test function aws made?

### DISCUSSION THREAD

**Discussion by GuiFlam** [Cyfrin/security-and-auditing-full-course-s23#113] [Q&A]
I am having difficulties understanding how the overflow test function was written? How was the 89 participants value was found? I am not sure to understand how I could have found this value on my own? Because I have tried with 100 participants and the test fails, there is no overflow, so I don't understand why it only works for specific values. 

Thanks

**Comment by cromewar** [comment]
Hello @GuiFlam can you please re-formulate a little bit, please check [this](https://github.com/Cyfrin/security-and-auditing-full-course-s23/blob/main/how-to-ask-a-question.md).
